app/services/idv/steps/verify_base_step.rb

```diff
@@ -49,10 +49,11 @@ def add_proofing_costs(results)
           if stage == :resolution
             # transaction_id comes from ConversationId
             add_cost(:lexis_nexis_resolution, transaction_id: hash[:transaction_id])
-            tmx_id = hash[:threatmetrix_request_id]
-            add_cost(:threatmetrix, transaction_id: tmx_id) if tmx_id
           elsif stage == :state_id
             process_aamva(hash[:transaction_id])
+          elsif stage == :threatmetrix
+            tmx_id = hash[:transaction_id]
```

Comment:
WDYT about adding a similar comment as above about where this field comes from?
```suggestion
            # transaction_id comes from request_id
            tmx_id = hash[:transaction_id]
```